addons/website/views/snippets/s_quotes_carousel.xml

```diff
@@ -13,7 +13,7 @@
             <!-- Content -->
             <div class="carousel-inner">
                 <!-- #01 -->
-                <div class="carousel-item active pt80 pb80" data-name="Slide">
+                <div class="carousel-item active oe_img_bg o_bg_img_center pt80 pb80" style="background-image: url('/web/image/website.s_carousel_default_image_1'); background-position: 50% 50%;" data-name="Slide">
```

Comment:
Use `s_quotes_carousel_demo_image_0` (because already 1 and 2) and create a new attachment for that, even if using the same image. That way it will be overridable independently by themes/industries.

@smottet Anything to do to add the new overridable image in your system?

Reply:
@qsm-odoo Well nothing to do on the odoo side but on the iap side it means either generating a new image for each of the 4000 industries either using an already validated image for this new snippet image.

Industries validation already started and I don't think we want to review 4000 industries each time we add a new snippet image... 

So the idea would be to use an image already used for an other snippet by adding a dict to the images list returned by get_images. The name would be website.s_quotes_carousel_demo_image_0 and the url, the url of an already validated image for the industry. We already have 25 differnett images for each industry. Is it a problem to use twice the same image?

![Screenshot from 2021-04-19 14-54-39](https://user-images.githubusercontent.com/49784121/115239895-7ee61c80-a11f-11eb-93f2-0169b033b1f1.png)

Reply:
@smottet Not sure to understand everything you said :grimacing: No problem to use an image multiple times, however it is not "perfect". We will create new snippets over the next years, I don't think it should be a problem to add new images to IAP. Maybe it could use another validated image as a fallback in case it has no validated one assigned yet but it should still be marked as an industry to (re)validate while there are unassigned images?

cc @JKE-be

Reply:
For the moment each industry has 25 images associated and they are currently being validated. These 25 images are used to override already existing snippet images.

The two solutions we have when we want to add a new snippet image is either generating 4000 new images (one for each industry) and manually validate them. Which is really time consuming. 

Either we can reuse one of the 25 images associated to the industry we already have. The disadvantage of this solution is that we will have the same image for different snippet images.

Reply:
The only problem in adding new images to IAP is the validation part which is really time consuming

Reply:
Imo it is the cost of this feature. But we need to pay it...

So yes, in case of new snippet, we should add new image on iap (with in worst case fallback on another existing picture) and so update your UI to validate it.

Reply:
Yes I understood that @smottet and same opinion as @JKE-be: yes a new snippet with a new image is a new image to validate on IAP side. Hence the fact we wanted an hierarchy of industry to make validation long-term viable :wink: 

Let's make it so we fallback on a validated image while we don't have a validated one for a new specific one (but still mark the industry as to update while we don't have a validated image for all images). Should we create a task for that? (no rush while this task is not merged) cc @sbuhl

Reply:
Ok in this case we will also have to make sure that we will have the human resources to make these validations.